lib/medium-client.js

```diff
@@ -0,0 +1,423 @@
+// Copright 2015 A Medium Corporation
```

Comment:
Happy to change - our style guide wasn't super explicit about this (other than saying '-' is preferred over '_')